changelog/2721.md

```diff
@@ -0,0 +1,3 @@
+### Fixed
+
+- add missing template for password_set view
```

Comment:
did u forget to add the template in the PR?

Reply:
ah :exploding_head: and now it's probably gone. Will check

Reply:
ah @m4ra the template is already merged, I just forgot the changelog in that PR so I added it here, sorry